easybuild/easyblocks/m/molpro.py

```diff
@@ -32,15 +32,31 @@
 import re
 import sys
 
+from easybuild.easyblocks.generic.binary import Binary
 from easybuild.easyblocks.generic.configuremake import ConfigureMake
+from easybuild.framework.easyblock import EasyBlock
+from easybuild.framework.easyconfig import CUSTOM
 from easybuild.tools.build_log import EasyBuildError
 from easybuild.tools.filetools import mkdir, read_file
-from easybuild.tools.run import run_cmd
+from easybuild.tools.run import run_cmd, run_cmd_qa
 
 
-class EB_Molpro(ConfigureMake):
+class EB_Molpro(Binary, ConfigureMake):
```

Comment:
swap this around; the first class being derived from determines what `super` calls refer to

```python
class EB_Molpro(ConfigureMake, Binary):
```